src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsServiceOptions.cs

```diff
@@ -1,29 +1,60 @@
 ﻿namespace Microsoft.ApplicationInsights.AspNetCore.Extensions
 {
+    using System.Reflection;
+    using Microsoft.Extensions.Logging;
+
     /// <summary>
     /// Application Insights service options defines the custom behavior of the features to add, as opposed to the default selection of featuers obtained from Application Insights.
     /// </summary>
     public class ApplicationInsightsServiceOptions
     {
         /// <summary>
+        /// Initializes a new instance of the <see cref="ApplicationInsightsServiceOptions" /> class.
         /// Application Insights service options that controlls the default behavior of application insights features.
         /// </summary>
         public ApplicationInsightsServiceOptions()
         {
             this.EnableQuickPulseMetricStream = true;
             this.EnableAdaptiveSampling = true;
+            this.LoggerMinimumLevel = LogLevel.Information;
```

Comment:
Does this play nice with the Microsoft.Extensions.Logging.Filter package? IMO setting just the LogLevel is almost never what you want - especially for remote loggers. People use ILogger for their own namespaces so it should be possible to configure separate log levels. 
If it doesn't work with the Filter package, how about extending it to accept a filter func, similar to the other existing loggers in M.E.Logging